ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -1025,15 +1024,16 @@ describe('WorkspaceEdit', () => {
 
     let billingDropDown = wrapper
       .find('[data-test-id="billing-dropdown"]')
-      .first()
-      .instance() as Dropdown;
+      .first();
 
     expect(mockEnsureBillingScope).toHaveBeenCalledTimes(0);
-    expect(billingDropDown.props.value).toEqual('billingAccounts/freetier');
-    expect(billingDropDown.props.options.map((o) => o.value)).toEqual([
+    expect(billingDropDown.props().value).toEqual('billingAccounts/freetier');
+    // @ts-ignore
```

Comment:
These ts-ignores are because options is not an attribute on the expected return type, but I believe that this is an Enzyme issue. I would say that we ignore these for now and convert these tests to React Testing Library.